### PR TITLE
Added WP To Readme 

### PR DESCRIPTION
**Pull request by EricFillion**
This branch contains the commits from #168, which will be ready to merge after fixing whitespace. 

Changes: 
1. README: updated the features section 
2. README: updated WP section 
3. README: removed WP finetuning section 
4. Added a file under  "examples/word_prediction" called "readme_examples.py." I used "word_prediction" rather than "wp" so that new users don't become confused if they're searching for wp examples.

### CODE REVIEW

**Comment by ted537** [comment]
Could you rebase such that we aren't trying to merge in the ALBERT changes twice?

**Comment by EricFillion** [comment]
> Could you rebase such that we aren't trying to merge in the ALBERT changes twice?



> Could you rebase such that we aren't trying to merge in the ALBERT changes twice?

We can just quickly merge #168. By "whitespaces are cool" do you mean to add or to remove whitespaces?

**Comment by ted537** [comment]
Ah true the same commits are in both there's no cherry picking nonsense here. Merging #168 first is a good plan

File: examples/word_prediction/readme_examples.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    happy_wp_albert = HappyWordPrediction("ALBERT", "albert-base-v2")
+    happy_wp_bert = HappyWordPrediction("BERT", "bert-base-uncased")
+
+
+def example_1_1():
+    happy_wp = HappyWordPrediction()  # default uses distilbert-base-uncased
+    result = happy_wp.predict_mask("I think therefore I [MASK]")
+    print(type(result))  # <class 'list'>
+    print(result)  # [WordPredictionResult(token_str='am', score=0.10172799974679947)]
+    print(type(result[0]))  # <class 'list'>
```
Comment (ted537) [inline examples/word_prediction/readme_examples.py:15]: by isnspection, `type(result[0])` should be `WordPredictionResult`

**Review by ted537 — CHANGES_REQUESTED** [review]

File: examples/word_prediction/readme_examples.py (hunk 1 above)
Comment (EricFillion) [reply examples/word_prediction/readme_examples.py:15]: Nice catch, I was copying and pasting stuff and forgot to change the comment.

**Review by ted537 — APPROVED** [review]

**Pull request merged**